### PR TITLE
Localise update-related process strings

### PR DESCRIPTION
**Pull request by Loreos7**

### CODE REVIEW

File: osu.Game/Localisation/NotificationsStrings.cs [hunk 1]
```diff
@@ -113,6 +113,28 @@ public static class NotificationsStrings
         /// </summary>
         public static LocalisableString MismatchingBeatmapForReplay => new TranslatableString(getKey(@"mismatching_beatmap_for_replay"), @"Your local copy of the beatmap for this replay appears to be different than expected. You may need to update or re-download it.");
 
+        /// <summary>
+        /// "You are now running osu! {version}.
+        /// Click to see what's new!"
+        /// </summary>
+        public static LocalisableString GameVersionAfterUpdate(string version) => new TranslatableString(getKey(@"game_version_after_update"), @"You are now running osu! {0}.
```
Comment (Loreos7) [inline osu.Game/Localisation/NotificationsStrings.cs:120]:
```suggestion
        public static LocalisableString GameVersionAfterUpdateToast(string version) => new TranslatableString(getKey(@"game_version_after_update_toast"), @"You are now running osu! {0}.
```

**Review by Loreos7 — COMMENTED** [review]
More clarity?

File: osu.Game/Localisation/NotificationsStrings.cs (hunk 1 above)
Comment (Joehuu) [reply osu.Game/Localisation/NotificationsStrings.cs:120]: Not really a toast (it's a notification and there are [`ToastStrings`](https://github.com/ppy/osu/blob/master/osu.Game/Localisation/ToastStrings.cs)), so no. I think what you had is fine as is.

**Review by Joehuu — APPROVED** [review]
Seems fine.

**Review by bdach — APPROVED** [review]

**Pull request merged**